autoload/neomake/makers/ft/scala.vim

```diff
@@ -1,11 +1,23 @@
 " vim: ts=4 sw=4 et
 function! neomake#makers#ft#scala#EnabledMakers()
-    let makers = ['scalac']
-    if neomake#utils#Exists('scalastyle')
+    let makers = get(g:, 'neomake_scala_fast', 0) ? ['fsc'] : ['scalac']
+    if neomake#utils#Exists('scalastyle') && get(g:, 'neomake_scala_style', 1)
         call add(makers, 'scalastyle')
+    endif
     return makers
 endfunction
 
+function! neomake#makers#ft#scala#fsc()
+    return {
+        \ 'args': [
+            \ '-Ystop-after:parser'
+        \ ],
+        \ 'errorformat':
+            \ '%E%f:%l: %trror: %m,' .
+            \ '%Z%p^,' .
+            \ '%-G%.%#'
+        \ }
+endfunction
```

Comment:
It's nice to have a single line of whitespace between functions for better readability.